CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [3.6.0] - 2019-08-30
```

Comment:
When a candidate release becomes final, I write something like:
`3.6.0-rc2 was considered final and became 3.6.0.` Otherwise, the section for the version is empty and looks like an error.